python/lsst/ts/envsensors/command_handler.py

```diff
@@ -213,15 +215,22 @@ async def _process_sensor_telemetry(self, telemetry):
         data = {"telemetry": telemetry}
         await self._callback(data)
 
-    def _get_device(self, configured_device):
+    def _get_device(self, configured_device: dict) -> typing.Optional[typing.Any]:
```

Comment:
I suspect you'll find that you have to be more specific than `dict` to make mypy happy. But that can wait until you run mypy on the code. Similarly for `list` as a type annotation above.

Reply:
Thanks for the suggestion. I'll do this in a later ticket.